src/styles/variables/colorPalette.js

```diff
@@ -13,3 +13,8 @@ export const textColors = {
 export const backgroundColors = {
   lightPink: '#F4EDED',
 };
+
+export const borderColors = {
+  default: '1px solid #DDDDDD',
+  primary: '1px solid #B02091',
```

Comment:
default and primary keys, this naming doesn't make a lot of sense. I would expect the default to be used app wide. When would I use `primary`